aiida/orm/implementation/general/group.py

```diff
@@ -133,6 +133,18 @@ def get_db_columns():
         """
         pass
 
+    @classmethod
+    def create(cls, *args, **kwargs):
+        """
+        Create a new group.
```

Comment:
Nit: Should this mention that "create" includes storing in the db? The name is a documented django convention but do we stick to it / have it documented anywhere?

Reply:
Updated help string to point out that it stores the group.